inception-imls-dl4j/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/dl4j/pos/DL4JSequenceRecommender.java

```diff
@@ -493,12 +493,17 @@ public EvaluationResult evaluate(List<CAS> aCas, DataSplitter aDataSplitter)
 
         int testSetSize = testSet.size();
         int trainingSetSize = trainingSet.size();
+        double trainRatio = (double) trainingSetSize / (double) (data.size() - testSetSize);
         
         if (trainingSetSize < 2 || testSetSize < 2) {
-            log.info("Not enough data to evaluate, skipping!");
+            String info = String.format(
+                    "Not enough training data: training set [%s] items, test set [%s] of total [%s].",
+                    trainingSetSize, testSetSize, data.size());
```

Comment:
You can directly log that with log.info and "{}" as place holders, no need for String.format.